app/src/main/java/com/example/calculator/mvp/model/CalculatorModel.java

```diff
@@ -1,12 +1,10 @@
 package com.example.calculator.mvp.model;
 
 import com.example.calculator.mvp.contract.CalculatorContract;
+import com.example.calculator.utils.ErrorEnum;
```

Comment:
Remember that we talked about change the name of ErrorUtils for ResultUtils and you can replace the NONE by a SUCCESS

Reply:
Refactored!